.github/workflows/sync-shared-config.yml

```diff
@@ -82,6 +82,7 @@ jobs:
           repository: ${{ matrix.repo }}
           path: target/${{ matrix.repo }}
           token: ${{ secrets.HOMEBREW_DOTGITHUB_WORKFLOW_TOKEN }}
+          persist-credentials: false
```

Comment:
Persisting credentials here was intentional - it's used by `git push` below.

Reply:
https://github.com/Homebrew/.github/pull/215